plugins/headlamp-plugin/bin/headlamp-plugin.js

```diff
@@ -28,7 +28,7 @@ function create(name, link) {
   const templateFolder = path.resolve(__dirname, '..', 'template');
   const indexPath = path.join(dstFolder, 'src', 'index.tsx');
   const packagePath = path.join(dstFolder, 'package.json');
-  const packageLockPath = path.join(dstFolder, 'package-lock.json');
+  // const packageLockPath = path.join(dstFolder, 'package-lock.json');
```

Comment:
Should we remove the lines instead of commenting them?

Reply:
Yop ok. Thanks.